examples/source/mnist_with_summaries/mnist_with_summaries.py

```diff
@@ -186,8 +186,9 @@ def feed_dict(train):
                     if is_chief:
                         train_writer.add_summary(summary, i)
             i += 1
-        train_writer.close()
```

Comment:
@joerg84 does this correct the issue of the workers hanging around?

Reply:
Not my catch :)